src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/finalparameters/InputFinalParametersPatternVariables.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Non compiled files should have comment to show in what version they becomes compiled.
See other files as examples, you will see how to make it jdk21